src/main/java/com/londonkoders/kwitter/user/UserController.java

```diff
@@ -22,5 +21,8 @@
     public Optional<User> getUser(@PathVariable Long id) {
         return userService.getUser(id);
     }
+
+    @PostMapping("/user")
+    public User createUser(@RequestBody User user) {return userService.createUser(user);}
```

Comment:
Can we format `return ...` in a new line?

Reply:
I was confused because all the existing methods there was a single line but it turned out it's just an auto-format in IDE ;)